### ISSUE TITLE
[request] Set the background color based on the image

### ISSUE THREAD

**Issue by hallgat89** [l3ib/nitrogen#99]
When selecting an image to be used, match the background color setting based on the image. This would be useful for small centered wallpapers. I currenty use a solution in a script which simply gets the pixel color at 0,0 (top left pixel) and sets is as a bg color. (The user would still be able to change it if needed)

**Comment by andoruB** [comment]
This would be a nice feature indeed.

**Comment by daf** [comment]
Yes, been on my ideas board for a long time.  Not sure when I'll get to it, but it's coming in the next one.
